acceptance/init_test.go

```diff
@@ -22,7 +22,7 @@ func TestInit(t *testing.T) {
 	r := runner.New(
 		"SOURCE="+srcDir,
 		"DESTINATION="+destDir,
-		"SHUTDOWN_DELAY=0",
+		"CIRCLECI_GOAT_SHUTDOWN_DELAY=0",
```

Comment:
We're now prefixing environment variables to easily identify ones associated with GOAT, so we can strip them from the task environment. This also helps prevent collisions with variables sharing common names.